test/playback-watcher.test.js

```diff
@@ -42,6 +42,97 @@ QUnit.module('PlaybackWatcher', {
   }
 });
 
+QUnit.test('skips over gap at beginning of stream if player is paused', function(assert) {
+  let vhsGapSkipEvents = 0;
+  let hlsGapSkipEvents = 0;
+
+  this.player.autoplay(true);
```

Comment:
I don't think this is necessary, as the events are triggered ourselves, and the case can still present without autoplay (if someone has a slow enough connection and is fast enough clicking the play button).

Reply:
Good point, will remove.